nucypher/cli/painting.py

```diff
@@ -532,7 +532,12 @@ def paint_stakes(emitter, stakeholder, paint_inactive: bool = False, staker_addr
                        pretty_fees,
                        min_reward_rate]
 
-        emitter.echo(f"\nStaker {staker.checksum_address} ════", bold=True, color='red' if missing else 'green')
+        line_width = 54
+        if staker.registry.source:
```

Comment:
Actors have a `network` attribute (see `actors.py`, L114). This is however something that's subject to whatever fixes #1580

Reply:
The conditional is to mitigate that in tests, there is no registry source and it is set to a constant instead.